### PR TITLE
bare metal and vSphere code block updates

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

File: modules/registry-configuring-storage-baremetal.adoc [hunk 1]
```diff
@@ -52,9 +53,14 @@ exporting *:/mnt/data
 +
```
Comment (jeana-redhat) [inline modules/registry-configuring-storage-baremetal.adoc:53]: (Same Q as for this content in the vSphere version)

**Review by jeana-redhat — COMMENTED** [review]
Just a couple questions about whether these changes apply to a specific block, but everything looked good :)

File: modules/registry-configuring-storage-vsphere.adoc [hunk 2]
```diff
@@ -60,9 +61,14 @@ exporting *:/mnt/data
 +
```
[the remark this answers is not in the document]
Comment (kalexand-rh) [reply modules/registry-configuring-storage-vsphere.adoc:61]: Good catch - I missed it in the note. Thank you!

**Pull request merged**